From: Michele Locati <michele@locati.it>
Date: Thu, 29 Nov 2018 16:44:12 +0100
Subject: [PATCH] Fix continue switch in Minifier

--- a/src/JShrink/Minifier.php
+++ b/src/JShrink/Minifier.php
@@ -181,7 +181,7 @@ class Minifier
                 // new lines
                 case "\n":
                     // if the next line is something that can't stand alone preserve the newline
-                    if (strpos('(-+{[@', $this->b) !== false) {
+                    if ($this->b !== false && strpos('(-+{[@', $this->b) !== false) {
                         echo $this->a;
                         $this->saveString();
                         break;
@@ -224,7 +224,7 @@ class Minifier
                             // check for some regex that breaks stuff
                             if ($this->a === '/' && ($this->b === '\'' || $this->b === '"')) {
                                 $this->saveRegex();
-                                continue;
+                                continue 3;
                             }
 
                             echo $this->a;
